index.d.ts

```diff
@@ -303,24 +334,31 @@ declare class Emittery {
 		});
 	```
 	*/
-	anyEvent(): AsyncIterableIterator<unknown>
+	anyEvent(): AsyncIterableIterator<
+		[keyof EventData, EventData[keyof EventData]]
+	>;
 
 	/**
 	Remove an `onAny` subscription.
 	*/
-	offAny(listener: (eventName: EventName, eventData?: unknown) => void): void;
+	offAny(
+		listener: <Event extends keyof EventData>(
+			eventName: Event,
```

Comment:
Shouldn't `Event` be `Name` for consistency?

Reply:
Yep, fixed